shell/yq.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#
+# Wrapper around `yq` that allows use of python-yq or gojq.
+# You can force using gojq by setting the environment variable YQ_USE_GOJQ=true.
+#
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+if [[ -z $GOJQ_VERSION ]]; then
+  GOJQ_VERSION="$(grep ^gojq: "$DIR"/../versions.yaml | awk '{print $2}')"
+fi
```

Comment:
```suggestion
GOJQ_VERSION="${GOJQ_VERSION:-$(grep ^gojq: "$DIR"/../versions.yaml | awk '{print $2}')}"
```

Easier to read for people who write a lot of bash. But also, I hate this fake yaml parsing you're doing here.

Reply:
> Easier to read for people who write a lot of bash.

I think I want to optimize for folks who don't read a lot of bash at this point, given the current number of reviewers with bash experience.

> I hate this fake yaml parsing you're doing here.

Well, there's a bootstrapping problem. Can't use `yq`/`gojq` on a YAML file if we don't have it installed yet.